README.md

```diff
@@ -7,12 +7,17 @@ This module spawns a thread and begins monitoring the process.
 
 It looks up /proc/* files on the system to report CPU Usage.
 It looks up /proc/pid/* files on the system to report its own stats.
-It calls the process.monitor.* methods to report total requests, open connections and total data transferred.
+It calls the process.monitor.* methods to report total requests, open connections and total
+data transferred.
+It attaches to the v8 garbage collection hooks to instrument (for each GC type)
+1.  `count` : number of times called in the past interval
+2.  `elapsed_ms`: total elapsed time nodejs thread is blocked
```

Comment:
Yes. On some unix-y systems there is a pseudo-filesystem called procfs.
http://en.wikipedia.org/wiki/Procfs